Sources/SWBCore/SpecImplementations/Tools/SwiftCompiler.swift

```diff
@@ -2132,7 +2132,7 @@ public final class SwiftCompilerSpec : CompilerSpec, SpecIdentifierType, SwiftDi
 
             // Compute the inputs and object output dependency paths.
             // Note that we compute the object file output paths here even if the compilation mode won't produce any, because these paths are used to compute the paths to other generated files.
-            // FIXME: If we want to match what Xcode did, then when using non-parallel WMO, we should include $(TARGET_NAME)-master.o as an output file, but not include the per-input-file object files as output files.
```

Comment:
Does Xcode actually call these primary these days?

Reply:
I dropped this comment because I think it's actually totally irrelevant now